### PR TITLE
CSIC-9-Snapshot-Added Unit test for rest client api

### PR DESCRIPTION
**Pull request by dhanashreebiradar**
Added unit test cases for rest client  api

### CODE REVIEW

File: api/nfsapiclient.go [hunk 1]
```diff
@@ -48,28 +48,28 @@ func (c *ClientService) OneTimeValidation(poolname string, networkspace string)
...
-		eResp, _ = apiresp.Result.(ExportResponse)
-	}
-	log.Info("Created export path : ", eResp.ExportPath)
-	return &eResp, nil
-}
+// func (c *ClientService) CreateExportPath(exportRef *ExportPathRef) (*ExportResponse, error) {
+// 	var err error
+// 	defer func() {
+// 		if res := recover(); res != nil && err == nil {
+// 			err = errors.New("CreateExportPath Panic occured -  " + fmt.Sprint(res))
```
Comment (gadekarnitesh) [inline api/nfsapiclient.go]: please remove the commented code

File: api/nfsapiclient.go [hunk 2]
```diff
@@ -244,29 +244,29 @@ func (c *ClientService) ExportFileSystem(export ExportFileSys) (*ExportResponse,
...
-	return &eResp, nil
-}
+// func (c *ClientService) GetExportByID(exportID int) (*ExportResponse, error) {
+// 	var err error
+// 	defer func() {
+// 		if res := recover(); res != nil && err == nil {
+// 			err = errors.New("GetExportByID Panic occured -  " + fmt.Sprint(res))
+// 		}
+// 	}()
+// 	log.Info("Get export path : ", exportID)
```
Comment (gadekarnitesh) [inline api/nfsapiclient.go]: please remove the commented code

**Review by Nilesh20 — APPROVED** [review]
Looks Good

**Review by gadekarnitesh — COMMENTED** [review]
please remove commented  code

File: api/nfsapiclient.go (hunk 1 above)
Comment (dhanashreebiradar) [reply api/nfsapiclient.go]: Fixed

File: api/nfsapiclient.go (hunk 2 above)
Comment (dhanashreebiradar) [reply api/nfsapiclient.go]: Fixed

**Pull request merged**